backend/benefit/calculator/tests/test_calculator_api.py

```diff
@@ -336,6 +336,83 @@ def test_application_edit_pay_subsidy_invalid_values(
     assert previous_data == data_after["pay_subsidies"]
 
 
+def test_application_edit_pay_subsidy_empty_date_values(
+    handler_api_client, handling_application
+):
+    data = HandlerApplicationSerializer(handling_application).data
+
+    previous_data = copy.deepcopy(data["pay_subsidies"])
```

Comment:
Naming here is inconsistent and makes understanding the code harder IMHO.

To make the naming consistent `previous_data` could be renamed to `previous_pay_subsidies` or alternatively `previous_data = copy.deepcopy(data)` could be used in place of `previous_data = copy.deepcopy(data["pay_subsidies"])`.